duktape/build.gradle

```diff
@@ -18,8 +18,15 @@ model {
     buildTypes {
       release {
         ndk {
-          CFlags.addAll(['-Os', '-fomit-frame-pointer'])
-          cppFlags.addAll(['-Os', '-fomit-frame-pointer'])
+          CFlags.addAll(['-Os', '-fomit-frame-pointer', '-DNDEBUG'])
+          cppFlags.addAll(['-Os', '-fomit-frame-pointer', '-DNDEBUG'])
+        }
+      }
+      debug {
```

Comment:
Not with the experimental plugin that we're currently using. It may not even be supported with it. I'll have to dig into it more.